src/rapidhtml/app.py

```diff
@@ -20,17 +47,26 @@ class RapidHTML(Starlette):
         - HTMX
     """
 
-    def __init__(self, *args, html_head: typing.Iterable = None, **kwargs) -> None:
+    def __init__(
+        self, *args, html_head: typing.Iterable = None, reload: bool = False, **kwargs
+    ) -> None:
```

Comment:
Probably time for a docstring
```suggestion
    ) -> None:
    """Initializes the RapidHTML application.

        Args:
            html_head (typing.Iterable, optional): Tags to inject into each 
                page's <head>. Defaults to None.
            reload (bool, optional): Enables live-reloading. Defaults to False.
    """
```